pkg/audit/manager.go

```diff
@@ -368,6 +368,7 @@ func (am *Manager) auditResources(
 	for gv, gvKinds := range clusterAPIResources {
 	kindsLoop:
 		for kind := range gvKinds {
+			log.Info("Listing objects for GVK", "group", gv.Group, "version", gv.Version, "kind", kind)
```

Comment:
Did we want this line and the following to actual use the manager log? `am.log`. Before we call `auditResources` we actually tag the audit manager log w the `auditID` which could be used to aggregate audit runs.

https://github.com/open-policy-agent/gatekeeper/blob/master/pkg/audit/manager.go#L165

Reply:
but that one defaults to always log. +1 on exposing verbosity so it doesn't flood the logs unless you need it.

Reply:
oops, yes, am.log() is a better call.

@ritazh I think logging/not logging is governed by the log level flag?